liby2util-r/src/include/y2util/Ustring.h

```diff
@@ -128,12 +128,7 @@ class Ustring
  {
 
   private:
-
-    /**
-     * !!! It should actualy be const !!!
-     * But that way default copy and assingment can be used.
-     **/
-    std::string _name;
+    const std::string* _name;
```

Comment:
yep, this is axiom of symbols. It is never released, same as in ruby. It lives forever.